db/migrate/20190311204446_add_fields_to_medication_reports.rb

```diff
@@ -0,0 +1,6 @@
+class AddFieldsToMedicationReports < ActiveRecord::Migration[5.1]
+  def change
+    add_column :medication_reports, :took_medication, :boolean
+    add_column :medication_reports, :not_taking_medication_reason, :string
```

Comment:
👍 - I think we'll be able to re-use this field to store the multiple-choice result for #54.